applications/launchpad_v2/src/layouts/MainLayout/styles.ts

```diff
@@ -0,0 +1,47 @@
+import { animated } from 'react-spring'
+import styled from 'styled-components'
+
+export const ScreenContainer = styled.div`
+  overflow: hidden;
+  border-radius: 10px;
+  flex: 1;
+`
+
+export const MainLayoutContainer = styled.div`
+  position: relative;
+  display: flex;
+  height: 100%;
+  flex: 1;
+  top: 0;
+  bottom: 0;
+  borderRadius: 10px;
+  overflow: hidden;
+`
+
+export const MainContainer = styled(animated.div)`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  padding-top: 60px;
+`
+
+export const ExpertViewBackgroundOverlay = styled(animated.div)`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  overflow: hidden;
+  background: #1a1a1a;
```

Comment:
theme

Reply:
Adding @todo in the file to catch up with another PR, because I cannot find anything matching in the colors and theme. Need to re-think how to catch background colors also for titlebar/navbar and expert view. Probably need to add new fields in `colors.ts` and `theme.ts`